.github/workflows/testing.yml

```diff
@@ -1,8 +1,9 @@
 name: Python package
 
 env:
-  DAKOTA_VERSION: 6.17.0
-
+  DAKOTA_VERSION: 6.18.0
+  DAKOTA_FILENAME: dakota-6.18.0-public-src-cli
```

Comment:
Is it possible to have one change when ever the version chnages? Here it is still two `DAKOTA_VERSION` and `DAKOTA_FILENAME`